_includes/px-k8s-prereqs.md

```diff
@@ -10,7 +14,7 @@ If you are installing Portworx 1.2, you *must* configure Docker to allow shared
 
 **Firewall**
 
-Ensure ports 9001-9015 are open between the nodes that will run Portworx. Your nodes should also be able to reach the port KVDB is running on (for example etcd usually runs on port 2379).
+Ensure network ports 9001 through 9015 are open between the nodes that will run Portworx. Your nodes should also be able to reach the port KVDB is running on (for example etcd usually runs on port 2379).
```

Comment:
this is not really true since the sdk we have been using ports 9001 - 9022
and if they change the ports it'll be start range + 18 ports